gstack-buffer/alloca/alloca.c

```diff
@@ -0,0 +1,16 @@
+#include <stddef.h>
+#include <stdint.h>
+
+#ifdef _MSC_VER
```

Comment:
why is it optional?

Reply:
Microsoft compiler does not support [VLA](https://en.wikipedia.org/wiki/Variable-length_array). But `char buffer[size]` is valid syntax for other compilers like clang and gcc